libs/wire-api/src/Wire/API/Routes/Public/Galley.hs

```diff
@@ -155,22 +155,25 @@ data Api routes = Api
         :> Post '[Servant.JSON] (Public.ConversationList Public.Conversation),
     -- This endpoint can lead to the following events being sent:
     -- - ConvCreate event to members
-    -- FUTUREWORK: errorResponse Error.notConnected
-    --             errorResponse Error.notATeamMember
-    --             errorResponse (Error.operationDenied Public.CreateConversation)
     getConversationByReusableCode ::
       routes
         :- Summary "Get limited conversation information by key/code pair"
+        :> CanThrow NotConnected
+        :> CanThrow OperationDenied
+        :> CanThrow NotATeamMember
         :> ZUser
         :> "conversations"
         :> "join"
         :> QueryParam' [Required, Strict] "key" Code.Key
         :> QueryParam' [Required, Strict] "code" Code.Value
         :> Get '[Servant.JSON] Public.ConversationCoverView,
-    -- FUTUREWORK: potential errors: codeNotFound, convNotFound, notATeamMember, convAccessDenied
     createGroupConversation ::
       routes
         :- Summary "Create a new conversation"
+        :> CanThrow NotATeamMember
+        :> CanThrow CodeNotFound
```

Comment:
Hm, isn't the CodeNotFound error thrown on a different endpoint? When requesting or trying to join a conversation which has a conversation code; rather than when creating a group conversation? Same for the ConvAccessDenied. I think the FUTUREWORK comments apply to the handle above the comment, not below the comment.

Maybe you want to swap the error list from createGroupConversation with the one from getConversationByReusableCode.

Reply:
Yes, you're right! I guess the comments went out of whack there after some refactoring. Having the list of exceptions in the type should make this sort of mistake a bit less likely.